virttest/qemu_vm.py

```diff
@@ -4270,6 +4279,13 @@ def migrate(self, timeout=virt_vm.BaseVM.MIGRATE_TIMEOUT, protocol="tcp",
                         raise exceptions.TestError(msg)
 
             logging.info("Migrating to %s", uri)
+            if (protocol in ('rdma', 'x-rdma', 'tcp', 'unix') and
+                    Flags.INCOMING_DEFER in self.devices.caps):
```

Comment:
Looks the following approach is more reasonable.
```python
deferral_incoming = False

    ...
    if Flags.INCOMING_DEFER in self.devices.caps:
        incoming_val = ' -incoming defer'
        deferral_incoming = True
    qemu_command += incoming_val

...

logging.info("Migrating to %s", uri)
if deferral_incoming:
    ...
```

Reply:
Thanks, have updated.